test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerChannelTests.cs

```diff
@@ -248,6 +260,19 @@ public void SendSendFunctionEnvironmentReloadRequest_SanitizedEnvironmentVariabl
             Assert.True(envReloadRequest.EnvironmentVariables["TestValid"] == "TestValue");
         }
 
+        [Fact]
+        public void SendFunctionEnvironmentReloadRequest_WithDirectory()
+        {
+            var environmentVariables = new Dictionary<string, string>()
+            {
+                { "TestValid", "TestValue" }
+            };
+
+            FunctionEnvironmentReloadRequest envReloadRequest = _workerChannel.GetFunctionEnvironmentReloadRequest(environmentVariables);
+            Assert.True(envReloadRequest.EnvironmentVariables["TestValid"] == "TestValue");
+            Assert.True(envReloadRequest.CurrentDirectory == _scriptRootPath);
+        }
+
```

Comment:
Also, take a look at tests here: https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests.Integration/Host/StandbyManager/StandbyManagerE2ETests_Windows.cs Please add a test for node.